cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -613,7 +621,7 @@ describe('Credentials Tabbed View - Job Templates', function () {
     cy.selectDropdownOptionByResourceName('inventory', awxInventory.name);
     cy.selectDropdownOptionByResourceName('project', `${(this.globalProject as Project).name}`);
     cy.selectDropdownOptionByResourceName('playbook', 'hello_world.yml');
-    cy.selectItemFromLookupModal('credential-select', machineCredential.name);
```

Comment:
We should revert this change. `selectItemFromLookupModal ` is the correct command.

Reply:
@kialam reverted.